modules/userId/index.js

```diff
@@ -423,5 +425,6 @@ init(config);
 // add submodules after init has been called
 attachIdSystem(pubCommonIdSubmodule);
 attachIdSystem(unifiedIdSubmodule);
+attachIdSystem(identityLinkSubmodule);
```

Comment:
The `attachIdSystem` should be used in the file `identityLinkSystem.js`, the `unifiedId` and `pubComonId` are the only submodules included by default, and they will also be optional starting with Prebid 3.0.

Reply:
Hi @idettman,
Thank you for the explanation. I'm changed implementation of identityLink submodule as you suggested. Follow the code in ID5 Submodule...